arduino/smartcar/secrets-default.hpp

```diff
@@ -0,0 +1,3 @@
+const auto MQTT_HOST = "hysm.dev";
+const auto MQTT_PORT = 1883;
+
```

Comment:
Remove file and merge with master to get the correct version of `secrets-default.hpp`